test/Java8andUp/src/org/openj9/test/reflect/GetMethodsTests.java

```diff
@@ -37,8 +37,8 @@
 import java.util.List;
 
 public class GetMethodsTests {
-
 	public static final Logger logger = Logger.getLogger(GetMethodsTests.class);
+	private final static boolean isJava8 = System.getProperty("java.specification.version").equals("8");  //$NON-NLS-1$//$NON-NLS-2$
```

Comment:
This test doesn't work, the java.specification.version is "1.8"

Reply:
Fixed.